cypress/features/regression/filterableSelect.feature

```diff
@@ -147,8 +147,8 @@ Feature: Filterable Select component
   @positive
   Scenario: Check the onKeyDown event after clicking arrow
     Given I open "Select filterable" component page "default story"
-    When I focus default Select input
-      And I click onto default select using "downarrow" key
+    When I click on controlled Select input
+      And I click onto controlled select using "downarrow" key
```

Comment:
Can this be amended to "press downarrow onto controlled select" or something similar rather than "click onto...using downarrow" to more accurately describe the action?

Reply:
how would you prefer to name it ?